plugins/ibm_qradar/help.md

```diff
@@ -619,468 +611,341 @@ Example input:
+|ID|integer|None|False|ID|None|
+|Is Deleted|boolean|None|False|Is deleted|None|
+|Is Reserved|boolean|None|False|Is reserved|None|
+|Text|string|None|False|text|None|
+  
+**note**
+
+|Name|Type|Default|Required|Description|Example|
+| :--- | :--- | :--- | :--- | :--- | :--- |
+|Created Time|integer|None|False|Created time|None|
+|ID|integer|None|False|ID|None|
+|Note Text|string|None|False|Note text|None|
+|Username|string|None|False|Username|None|
+  
+**offense**
+
+|Name|Type|Default|Required|Description|Example|
+| :--- | :--- | :--- | :--- | :--- | :--- |
+|Assigned To|string|None|False|Assigned to|None|
+|Categories|[]string|None|False|Categories|None|
+|Category Count|integer|None|False|Category count|None|
+|Close Time|integer|None|False|Close time|None|
+|Closing Reason ID|integer|None|False|Closing reason ID|None|
+|Closing User|string|None|False|Closing user|None|
+|Credibility|integer|None|False|Credibility|None|
+|Description|string|None|False|Description|None|
+|Destination Networks|[]string|None|False|Destination networks|None|
+|Device Count|integer|None|False|Device count|None|
+|Domain ID|integer|None|False|Domain ID|None|
+|Event Count|integer|None|False|Event count|None|
+|First Persisted Time|integer|None|False|First persisted time|None|
+|Flow Count|integer|None|False|Flow count|None|
+|Follow Up|boolean|None|False|Follow up|None|
+|ID|integer|None|False|ID|None|
+|Inactive|boolean|None|False|Inactive|None|
+|Last Persisted Time|integer|None|False|Last persisted time|None|
+|Last Updated Time|integer|None|False|Last updated time|None|
+|Local Destination Address IDs|[]integer|None|False|Local destination address IDs|None|
+|Local Destination Count|integer|None|False|Local destination count|None|
+|Log Sources|[]log_sources|None|False|Log sources|None|
+|Magnitude|integer|None|False|Magnitude|None|
+|Offense Source|string|None|False|Offense source|None|
+|Offense Type|integer|None|False|Offense type|None|
+|Policy Category Count|integer|None|False|Policy category count|None|
+|Protected|boolean|None|False|Protected|None|
+|Relevance|integer|None|False|Relevance|None|
+|Remote Destination Count|integer|None|False|Remote destination count|None|
+|Rules|[]rules|None|False|Rules|None|
+|Security Category Count|integer|None|False|Security category count|None|
+|Severity|integer|None|False|Severity|None|
+|Source Address IDs|[]integer|None|False|Source address IDs|None|
+|Source Count|integer|None|False|Source count|None|
+|Source Network|string|None|False|Source network|None|
+|Start Time|integer|None|False|Start time|None|
+|Status|string|None|False|Status|None|
+|Username Count|integer|None|False|Username count|None|
 
 
 ## Troubleshooting
-
-_This plugin does not contain any troubleshooting information._
+  
+*There is no troubleshooting for this plugin.*
 
 # Version History
 
+* 1.1.0 - Adding in new connection test | Updated the SDK to version 5.4.4 | Ran refresh to upadate all schemas and unit tests
```

Comment:
Typo in update
Can you update to using sdk v5.4.5?

Reply:
yeah the version 5.4.5 will be added to this plugin